Bayesian_Cognitive_Modeling/ParameterEstimation/DataAnalysis/Correlation_2.stan

```diff
@@ -0,0 +1,44 @@
+// #### Notes to Stan model #######################################################
+// ## 1) All notes from previous model Correlation_1 also apply to this model.
+// ## 2) If you change sigmaerror to c(0.03, 10) as suggested in excercise 5.2.2
+// ##    warning statements will be more frequent and posterior less smooth.
+// ################################################################################
+
+// Pearson Correlation With Uncertainty in Measurement
+data { 
+  int<lower=0> n;
+  array[n] vector[2] x;
+  vector[2] sigmaerror;
+}
+
+parameters {
+  vector[2] mu;
+  vector<lower=0>[2] lambda;
+  real<lower=-1, upper=1> r;
```

Comment:
You can include a note that there's an implicit uniform prior on `r`, but I'd rename to `rho` to stick to the usual notation for correlation and the usual notation of Greek letters for parameters.